his/core/application-integration-importhostdefs.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Importing COBOL Host Definitions"
+description: Importing COBOL Host Definitions for CICS, IMS and Host Files
+author: haroldcampos
+ms.author: "hcampos"
+ms.prod: "host-integration-server"
+ms.topic: how-to
+ms.date: "10/17/2023"
+---
+
+# Importing COBOL Host Definitions
+
+When the original Host Integration Server TI Designer feature was introduced, it was provided with a full set of capabilities to design metadata artifacts from scratch with the assistance of a Mainframe Programmer. The ability to import Host Definitions in the format of a COBOL or RPG copybook was introduced to support scenarios to automate design and to reduce involvement of Mainframe Programmers. Over the years, this became the principal option to create Metadata artifacts.
+
+## Prerequisites
+
+The Main prerequisite is to obtain the Host Definitions (copybooks) that will be imported in the HIS Designer for Logic Apps. The HIS Designer for Logic Apps supports COBOL and RPG copybooks.
+It is also important to understand the Programming model for the technology to integrate against (CICS or IMS). Both platforms have different requirements and ways to pass and receive information. Please get yourself familiar with the appropriate [Programming Model](choosing-the-appropriate-programming-model1.md) before importing the Host Definitions.
+
+## Preparing a COBOL Copybook
+
+- COBOL copybooks should follow the basic COBOL coding rules. The HIS Designer for Logic Apps enforces many of those. The main ones are the following:
+
+   |Columns  |Type  |Observation  |
+   |---------|---------|---------|
+   |1–6     |     Sequence number    |   The Mainframe programmer should not enter anything in these positions.      |
+   |7     |      Indicator   |    This can be used to code a comment (*). A slash (/) is also accepted.     |
+   |8-11     |   A Margin (Area A)      |  77 level numbers and 01 level numbers.       |
+   |12-72     |   B Margin (Area B)      |  These are reserved for 02 levels and upper.       |
+   |73-80     |   Identification      |    No definitions are allowed here.     |
+
+- Verify that the COBOL or RPG copybook meets the requirement of the selected [Programming Model](choosing-the-appropriate-programming-model1.md).
+- Verify the dots at the end of every line. Although with the COBOL latest versions this is not required, be prepared.
```

Comment:
Be prepared for what?

```suggestion
- Verify the dots at the end of every line. Make sure they appear, even though COBOL's latest versions don't require this formatting.
```

Reply:
@haroldcampos: Could you please clarify the original wording or confirm this rewrite? Thanks!